CHANGELOG.md

```diff
@@ -2,6 +2,11 @@
 
 All notable changes to the Zowe CLI package will be documented in this file.
 
+## Recent Changes
+
+- Make username, password, and host optional on z/OSMF profiles and update profile creation doc to reflect the change.
+- Create z/OSMF profile creation system tests.
```

Comment:
Would suggest removing the second line here since it's not a user-facing change, and to be consistent with our practice that if a PR only changes tests, it doesn't warrant a version bump or changelog entry.